influxdb/Makefile

```diff
@@ -69,7 +69,7 @@ push: gcr-login $(addprefix sub-push-,$(ALL_ARCHITECTURES))
 
 sub-push-%:
 	$(MAKE) ARCH=$* PREFIX=$(PREFIX) VERSION=$(VERSION) build
-	docker push $(PREFIX)/heapster-influxdb-$(ARCH):$(VERSION)
+	docker push $(PREFIX)/heapster-influxdb-$*:$(VERSION)
```

Comment:
Good catch! And glad to see that i have learned another info about `make`. 

The link below is just for any user interesting in this to consult: https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html